iclr2017/python/get-workshop-submissions.py

```diff
@@ -36,13 +36,14 @@
     if ext.lower()=='csv':
         with open(args.output, 'wb') as outfile:
             csvwriter = csv.writer(outfile, delimiter=',')
-            fieldnames = ['id', 'number', 'date', 'title', 'TL;DR', 'abstract','keywords','authors','authorids','conflicts']
+            fieldnames = ['id', 'number', 'original', 'date', 'title', 'TL;DR', 'abstract','keywords','authors','authorids','conflicts']
             csvwriter.writerow(fieldnames)
 
             for note in notes:
                 row = []
                 row.append(note.id)
                 row.append(note.number)
+                row.append('https://openreview.net/forum?id=%s' % note.original if note.original else None)
```

Comment:
if note.original is None, then the field looks like this:

https://openreview.net/forum?id=None